lib/mock.js

```diff
@@ -23,7 +23,7 @@ function walk (dir) {
 	var list = fs.readdirSync(dir)
 
 	list.forEach(function (file) {
-		file = dir + '/' + file
+		file = dir + path.sep + file
```

Comment:
nit: using `join` makes it more readable

Reply:
fine with keeping it as is also, this PR looks good

Reply:
Gonna leave it as is since it's closer to the original.